app/controllers/ems_cloud_controller.rb

```diff
@@ -160,7 +160,7 @@ def ems_cloud_form_fields
                      :provider_id                     => @ems.provider_id ? @ems.provider_id : "",
                      :hostname                        => @ems.hostname,
                      :api_port                        => @ems.port,
-                     :api_version                     => @ems.api_version,
+                     :api_version                     => @ems.api_version ? @ems.api_version : "v2",
```

Comment:
So I'm a little confused...This code in question is in the Rails side and you have the `@ems`, so not sure where angular fits in.  If you use default_value_for in the model for api_version, then this code should work as it was.